book.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'item'
+require 'securerandom'
+
+class Book < Item
+  attr_accessor :author, :publisher, :cover_state, :publish_date, :label
+  attr_reader :id
+
+  def initialize(_id, author, publisher, cover_state, publish_date, label)
+    super()
+    @id = generate_random_id
+    @author = author
+    @publisher = publisher
+    @cover_state = cover_state
+    @publish_date = publish_date
+    @label = label
+  end
+
+  def generate_random_id
+    SecureRandom.uuid
+  end
+
+  def can_be_archived?
+    if @cover_state == bad || super
```

Comment:
:red_circle: [Requested] In Ruby, there is a difference between bad and 'bad'. When you use bad without quotes, it is treated as a variable or identifier, whereas 'bad' with single or double quotes is a string. Keep in mind that variables and strings are distinct in Ruby, and using the appropriate one in your code is important for correct behaviour. Kindly make sure you replace bad with` 'bad'`

Reply:
Hi @bienvenuushindi 
I have updated book.rb and surrounded bad with quotes 'bad'.
Please kindly review and approve if all is in order.Because the system has already moved our team to the next project.
Thank you

Reply:
hi @bienvenuushindi 
I have update book.rb cover state to:
`if @cover_state == 'bad' || super`